bittensor/executor.py

```diff
@@ -61,7 +61,7 @@ def __init__(
         self.wallet = wallet
 
         if subtensor == None:
-            subtensor = bittensor.subtensor.Subtensor( config = self.config, wallet = self.wallet )
+            subtensor = bittensor.subtensor.Subtensor( config = self.config )
```

Comment:
Is there a reason to assume subtensor == None?

Reply:
You can instantiate the executor with a subtensor client, otherwise it just uses the config file.